lib/elixir/pages/Compatibility and Deprecations.md

```diff
@@ -99,21 +99,21 @@ Version | Deprecated feature                                  | Replaced by (ava
 [v1.8]  | Returning `{:ok, contents}` or `:error` from `Mix.Compilers.Erlang.compile/6`'s callback | Return `{:ok, contents, warnings}` or `{:error, errors, warnings}` (v1.6)
 [v1.7]  | `Code.get_docs/2`                                   | `Code.fetch_docs/1` (v1.7)
 [v1.7]  | Calling `super/1` on GenServer callbacks            | Implenting the behaviour explicitly without calling `super/1` (v1.0)
-[v1.7]  | `Enum.chunk/2`[`/3/4`](`Enum.chunk/4`)              | `Enum.chunk_every/2`[`/3/4`](`Enum.chunk_every/4`) (v1.5)
+[v1.7]  | `Enum.chunk/2`[`,3,4`](`Enum.chunk/4`)              | `Enum.chunk_every/2`[`,3,4`](`Enum.chunk_every/4`) (v1.5)
```

Comment:
Oh, now that the syntax is officially supported, we can write this as:

```suggestion
[v1.7]  | `Enum.chunk/2,3,4`              | `Enum.chunk_every/2,3,4` (v1.5)
```

Can you update the examples below?